src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private Random randomChoice = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+    private Figure[] figures = {new IsoscelesTrapezoid(getRandomNumber(),
+            getRandomNumber(), getRandomNumber(), randomColor.getRandomColor()),
+            new Circle(getRandomNumber(), randomColor.getRandomColor()),
+            new RightTriangle(getRandomNumber(), getRandomNumber(), randomColor.getRandomColor()),
+            new Rectangle(getRandomNumber(), getRandomNumber(), randomColor.getRandomColor()),
+            new Square(getRandomNumber(), randomColor.getRandomColor())};
+
+    public int getRandomChoice() {
+        return randomChoice.nextInt(figures.length);
+    }
+
+    public int getRandomNumber() {
+        final int maxValue = 50;
+        return random.nextInt(maxValue);
+    }

```

Comment:
FigureSupplier should have only two public methods(getRandomFigure, getDefaultFigure)
```suggestion
```

Reply:
not fixed